idseq_pipeline/commands/common.py

```diff
@@ -145,12 +144,13 @@ def __enter__(self):
 
     def __exit__(self, *args):
         self.timer_thread.cancel()
+        self.exited = True
 
 
 class CommandTracker(Updater):
 
-    lock = threading.RLock()
-    count = 0
+    lock = multiprocessing.RLock()
+    count = multiprocessing.Value('i', 0)
```

Comment:
This counter needs to be incremented from multiple processes when they all execute commands.   This requires it to be wrapped in a special container, called multiprocessing.Value.